doc/api/events.md

```diff
@@ -1080,7 +1086,7 @@ Neither the `EventTarget` nor `Event` classes are available for end
 user code to create.
```

Comment:
This sentence is no longer true.
```suggestion
```

I’m fixing that in <https://github.com/nodejs/node/pull/37059>.